### ISSUE TITLE
Change Meeting Type

### ISSUE THREAD

**Issue by jpbraun** [SergeyMosin/Appointments#140]
Due to the unpredictability of the corona pandemic, our parents must be given the opportunity to change the type of meeting ("Online (Audio/Video)" or "In-person-meeting) at short notice. Please add to the confirmation email a link for changing the appointment type (deleting the appointment and creating a new appointment at the same date and time but with the opposite meeting type)?

**Comment by SergeyMosin** [comment]
Available in v1.7.9
``Settings > Talk Integration > Add "Meeting Type" form field > Type change email text``
_(NC app store server is still down, I'll upload when it's back online)_

If this field is not empty and has two tokens ``{{link_text}}`` _(can contain any text)_ and ``{{new_type}}`` _(MUST be new_type)_, then this text will be attached to the email and attendees will be able to switch their meeting type simply by clicking the link ``{{link_text}}``.

For example, this:
```
Click {{here}} to change your appointment type to {{new_type}}.
```
will look similar to this:

------

Click [here](url) to change your appointment type to Online (audio/video).

------

Talk rooms will be created and deleted  automatically when a meeting type changes.

![meeting-type](https://user-images.githubusercontent.com/10747559/95678427-88079700-0b9a-11eb-9989-f491ac752bd6.jpg)

**Comment by jpbraun** [comment]
Hi Sergey,
thank you very much for your quick response. Your App is a great addition to Nexcloud!
Jan
